### DISCUSSION TITLE
Could you "re-close" https://github.com/python-gitlab/python-gitlab/issues/2249 as "Not planned"?

### DISCUSSION THREAD

**Discussion by stdedos** [python-gitlab/python-gitlab#2992] [Ideas]
I guess, since you are "distancing yourself" from the logic of Gitlab's output, you are not planning to "fix" this 😅

**Comment by nejch** [comment]
@stdedos I'm not sure GitHub even distinguished between `Closed as completed` / `Closed as not planned` back in 2022 when this was discussed, but if it's something you feel strongly about we can do it sure.

But otherwise, not sure what you mean by "distancing ourselves" here? I had a quick re-read and one of them is a project setting (default branch) and the other is actual, existing branches, which I guess could lead to some discrepancies if GitLab allows setting these independently, but that is simply what the API returns. Is this an issue for python-gitlab specifically?

**Reply by stdedos** [reply]
> @stdedos I'm not sure GitHub even distinguished between `Closed as completed` / `Closed as not planned` back in 2022 when this was discussed, ...

Oops 😅 I didn't notice that

> ... but if it's something you feel strongly about we can do it sure.

Not so much, no (esp. after hearing the above) - but I did happened to get there, and I tend to interpret purple as "what the title says was a problem, and it was fixed and I don't need to really read it unless I'm really digging it".

> But otherwise, not sure what you mean by "distancing ourselves" here? ... but that is simply what the API returns. Is this an issue for python-gitlab specifically?

That is exactly what I mean. If the API does that, then it does that. You are just an API wrapper. So "all these" are better expressed by the "Close as not planned. Won’t fix, can’t repro, duplicate, stale" state (i.e. *Not planned*)

-----

Either way, (again) thank you for taking the time @nejch 🙏
